models/reviews.model.js

```diff
@@ -92,8 +135,8 @@ exports.fetchReviewCommentsById = (id) => {
 };
 
 exports.addCommentByReviewId = (reviewId, newComment) => {
-  if (Object.keys(newComment).length < 2){
-    return Promise.reject({status:400, msg: 'Bad Request'})
+  if (Object.keys(newComment).length < 2) {
```

Comment:
Id prefer you check for the exact keys you are after here. This could be 2 completely different and incorrect keys and still pass